CRM/Queue/DAO/Queue.php

```diff
@@ -56,13 +56,49 @@ class CRM_Queue_DAO_Queue extends CRM_Core_DAO {
   public $type;
 
   /**
-   * Should the standard background attempt to autorun tasks in this queue?
+   * Name of the task runner
    *
-   * @var bool|string|null
-   *   (SQL type: tinyint)
+   * @var string
+   *   (SQL type: varchar(64))
    *   Note that values will be retrieved from the database as a string.
    */
-  public $is_autorun;
+  public $runner;
+
+  /**
+   * Maximum number of items in a batch. Tip: If you expand batch_limit, then also consider expanding lease_time.
+   *
+   * @var int|string
```

Comment:
outside the scope of this PR - but batch_limit is NOT int|string - it is int and " Note that values will be retrieved from the database as a string" is not true - this is coming from gencode from the looks

Reply:
Yeah, that is confusing.

FWIW, I modeled the XML for this field after `Event.xml`'s `max_participants` (which seemed analogous). The `max_participants` has the same issue (ie it lists `string` as a type even though it's really only for numeric data).